source/whats-new.txt

```diff
@@ -20,6 +21,47 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.7:
+
+What's New in 4.7
+-----------------
+
+New features of the 4.7 driver release include:
+
+- Improved change stream support
+
+  - Enhanced pre-image and post-image support for :ref:`change streams <java-usage-watch>`.
+    Change stream watch helpers now accept ``whenAvailable`` and ``required``
+    for the fullDocument option.
+  - Added a new ``fullDocumentBeforeChange`` option, which accepts
+    ``whenAvailable`` and ``required``.
+  - Change events may now include a ``fullDocumentBeforeChange`` response field.
+  - Added ``showExpandedEvents`` support for change streams.
+  - Added ``wallTime`` support in ``ChangeStreamDocument`` class.
+
+- Clustered index creation support
+- The driver supports several new features related to :v6.0:`Queryable Encryption </core/queryable-encryption/>`,
```

Comment:
Suggestion: 1) Keep the structure of the bulleted points consistent with the rest of the bulleted items. 2) Driver is already the implied subject. "Support for several new Queryable Encryption features."

Suggestion: v6.0 directive is no longer needed since the Manual v6.0 has been deployed (earlier today)